pkg/sql/show_syntax.go

```diff
@@ -126,7 +126,7 @@ func runShowSyntax(
 		}
 	} else {
 		for _, stmt := range stmts {
-			if err := report(ctx, "sql", tree.AsStringWithFlags(stmt, tree.FmtParsable)); err != nil {
+			if err := report(ctx, "sql", tree.Pretty(stmt)); err != nil {
```

Comment:
Ouch, that's not going to work. `Pretty` currently includes `FmtShowPassword`, which SHOW SYNTAX should not include.

Also it seems like it's odd to provide this behavior without also making it configurable.

I kindly invite you to consider adding https://github.com/mjibson/cockroach/pull/8 on top of this.